src/shared/analytics/analytics.background.ts

```diff
@@ -106,12 +106,14 @@ function trackAppEvents({ account }: { account: Account }) {
   });
 
   emitter.on('buttonClicked', (data) => {
+    const { buttonName, buttonScope, pathname, walletAddress } = data;
     const params = createParams({
-      request_name: 'screen_view',
-      screen_name: data.pathname,
-      button_name: data.buttonName,
+      request_name: 'button_clicked',
+      screen_name: pathname,
+      button_name: buttonName,
+      wallet_address: walletAddress,
     });
-    const event_name = `${data.buttonScope}: Button Pressed`;
+    const event_name = `${buttonScope}: Button Pressed`;
```

Comment:
Why is it "pressed" here and "clicked" above? :)

Reply:
I think, this is just a typo. I also not sure that we need aligh this naming with event naming. In our code we use 'click' rather then 'press'